styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -347,16 +349,17 @@ private boolean shouldExecute(RunState runState) {
     return !deadline.isAfter(now);
   }
 
-  private void sendDequeue(InstanceState instanceState, Set<String> resourceIds) {
-    final WorkflowInstance workflowInstance = instanceState.workflowInstance();
-    final RunState state = instanceState.runState();
+  private void sendDequeue(InstanceState instance, Set<String> resourceIds) {
```

Comment:
Now let's ask the question here. Why renaming? `instanceState` feels more precise.